packages/flutter_tools/test/general.shard/plugins_test.dart

```diff
@@ -315,6 +421,31 @@ dependencies:
       }, overrides: <Type, Generator>{
         FileSystem: () => fs,
         ProcessManager: () => FakeProcessManager.any(),
+        SystemClock: () => mockClock,
+        FlutterVersion: () => mockVersion
+      });
+
+      testUsingContext('No changes to the plugin list does not invalidate the Cocoapod lockfiles', () {
```

Comment:
This tests the new change, where podfiles shouldn't be invalidated if plugins didn't change